src/Components/Auth/Profile/Profile.jsx

```diff
@@ -0,0 +1,69 @@
+import axios from "axios";
+import { useEffect } from "react";
+import { useAuth } from "../../../Context";
+import { VIZZ_API } from "../../../utils";
+import { FaUserCircle } from "react-icons/fa";
+import "./styles.css";
+
+export const Profile = () => {
+	const {
+		state: { userDetails, token },
+		dispatch,
+		logout,
+	} = useAuth();
+
+	useEffect(() => {
+		if (!userDetails) {
+			(async () => {
+				try {
+					const {
+						data: { response },
+						status,
+					} = await axios({
+						url: `${VIZZ_API}/users/info`,
+						method: "GET",
+						headers: {
+							Authorization: `Bearer ${token}`,
+						},
+					});
+					console.log({ response });
+
+					if (status === 200) {
```

Comment:
I don't think you need to check for the status here.. If the status is something other than 2xx then it'll automatically go to the error handler.